core/src/main/java/org/elasticsearch/index/shard/ShardPath.java

```diff
@@ -168,34 +161,6 @@ public static void deleteLeftoverShardDirectory(ESLogger logger, NodeEnvironment
         }
     }
 
-    /** Maps each path.data path to a "guess" of how many bytes the shards allocated to that path might additionally use over their
-     *  lifetime; we do this so a bunch of newly allocated shards won't just all go the path with the most free space at this moment. */
-    private static Map<Path,Long> getEstimatedReservedBytes(NodeEnvironment env, long avgShardSizeInBytes, Iterable<IndexShard> shards) throws IOException {
```

Comment:
@mikemccand this is unused and get removed in 4d38856f7017275e326df52a44b90662b2f3da6a - was this a mistake or did you just not remove this method?